src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Figure implements FigureInfo {
+    private String color;
+    private String name;
```

Comment:
remove this field, you don't need it, if you want write name of figure in method that responsible for drawing directly